tslearn/metrics.py

```diff
@@ -9,6 +9,7 @@
     _jacobian_product_sq_euc
 from sklearn.metrics.pairwise import euclidean_distances
 from numba import njit, prange
+from sklearn.externals.joblib import Parallel, delayed
```

Comment:
joblib will be vendored from scikit-learn in a future release (0.23 or 0.24 I think), and is already a dependency of scikit-learn. It would be better to add it as a dependency and to import from the original package instead of the version saved in scikit-learn.